subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/SymbolHider.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.language.swift.tasks;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+
+/**
+ * Parse and hide given symbols in on object file based on COFF format documented
+ * here: https://docs.microsoft.com/en-us/windows/desktop/debug/pe-format
+ */
+public class SymbolHider {
+    DataReader data;
+    private byte[] bytes;
+
+    class DataReader {
```

Comment:
We should probably just make this a private class for now.   And make it static.

Reply:
done